resources/views/pages/Hub/Missions/Missions.svelte

```diff
@@ -82,7 +82,7 @@
         on:cardClicked={handleCardClicked}
     />
     <MissionCollection
-        title={"Played missions"}
+        title={"arcomm"}
```

Comment:
Change left over from testing

Reply:
reverted change on this back to the previous line